README.md

```diff
@@ -18,10 +20,18 @@ Prerequisites: A GitHub account (for logging into Gitpod)
 * After opening the tool via the notification, the interface loads in a few seconds.
 * Now you're ready to try the trace viewer!
     * Head to the trace viewer tab in the left side menu to get started.
-    * The tool is already loaded with example traces from a set of [Trace Visualisation Labs](https://github.com/tuxology/tracevizlab), so no need to hunt for your own.
+    * The tool is loaded with the example traces from a set of [Trace Visualisation Labs](https://github.com/tuxology/tracevizlab). To analyse your own traces, [download the application](https://github.com/theia-ide/theia-trace-extension#download-the-application) for Linux.
```

Comment:
Proposed correction: "download an application" instead of "download the application"

This implies that there are other ways to include the extension in an application (rather than the only way).